server/channels/store/sqlstore/post_store.go

```diff
@@ -2780,6 +2855,23 @@ func (s *SqlPostStore) GetDirectPostParentsForExportAfter(limit int, afterId str
 			*post.ChannelMembers = channelMembersMap[channelId]
 		}
 	}
+
+	flaggedMap, err := s.getFlaggedPostUsernames(postIds)
```

Comment:
I notice this is repeated. Maybe it's worth a common function.

Reply:
Removed this one too